File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        
        if(s.length()%2 == 0){
            int a = (s.length()/2)-1;
            int a1 = s.length()/2;
            answer += s.charAt(a);
            answer += s.charAt(a1);
        }else{
            int a = s.length()/2;
            answer += s.charAt(a);
        }
            
       
        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    if(s.length%2===0){
        answer+=s[s.length/2-1]
        answer+=s[s.length/2]
    }else if(s.length%2==1){
        answer+=s[Math.floor(s.length/2)]
    }
    return answer;
}